### PR TITLE
Code cleanup of Dagr module

### PR DESCRIPTION
**Pull request by jokoho48**

### CODE REVIEW

File: addons/dagr/functions/fnc_outputData.sqf [hunk 1]
```diff
@@ -68,7 +69,7 @@ GVAR(outputPFH) = [{
     __gridControl ctrlSetText format ["%1", _dagrGrid];
     __speedControl ctrlSetText format ["%1", _dagrSpeed];
     __elevationControl ctrlSetText format ["%1", _dagrElevation];
-    __headingControl ctrlSetText (if (!GVAR(useDegrees)) then { format ["%1", _dagrHeading] } else { format ["%1 �", _dagrHeading] });
+    __headingControl ctrlSetText (if (!GVAR(useDegrees)) then { format ["%1", _dagrHeading] } else { format ["%1 �", _dagrHeading] });
```
Comment (Glowbal) [inline addons/dagr/functions/fnc_outputData.sqf]: Wierd character symbol here

File: addons/dagr/functions/fnc_outputVector.sqf [hunk 2]
```diff
@@ -94,5 +93,5 @@ GVAR(vectorGrid) = _dagrGrid;
 __gridControl ctrlSetText format ["%1", _dagrGrid];
 __speedControl ctrlSetText format ["%1", _dagrDist];
 __elevationControl ctrlSetText format ["%1", _dagrElevation];
-__headingControl ctrlSetText (if (!GVAR(useDegrees)) then { format ["%1", _bearing] } else { format ["%1�", _bearing] });
+__headingControl ctrlSetText (if (!GVAR(useDegrees)) then { format ["%1", _bearing] } else { format ["%1�", _bearing] });
```
Comment (Glowbal) [inline addons/dagr/functions/fnc_outputVector.sqf]: Also a wierd character symbol here.

File: addons/dagr/functions/fnc_outputWP.sqf [hunk 3]
```diff
@@ -79,12 +80,12 @@ GVAR(outputPFH) = [{
...
     // WP Heading
     _bearing = floor ((_WPpos vectorDiff _MYpos) call CBA_fnc_vectDir);
-    
+
     // Output
     __gridControl ctrlSetText format ["%1", _dagrGrid];
     __speedControl ctrlSetText format ["%1", _bearing];
     __elevationControl ctrlSetText format ["%1", _dagrGrid2];
-    __headingControl ctrlSetText (if (!GVAR(useDegrees)) then { format ["%1", _dagrHeading] } else { format ["%1�", _dagrHeading] });
+    __headingControl ctrlSetText (if (!GVAR(useDegrees)) then { format ["%1", _dagrHeading] } else { format ["%1�", _dagrHeading] });
```
Comment (Glowbal) [inline addons/dagr/functions/fnc_outputWP.sqf]: Again, here as well

**Comment by Glowbal** [comment]
Apart from the character symbol, this is :+1:

**Comment by jokoho48** [comment]
cant fix this everytime i try to change this its back sorry

**Comment by jonpas** [comment]
Fix your encoding, or just write "degrees".

**Comment by PabstMirror** [comment]
I think it's an encoding thing, we can just merge and fix on master.

**Pull request merged**